packages/cms/pages/resources/edit/edit.tsx

```diff
@@ -0,0 +1,353 @@
+import React, { useState } from 'react'
```

Comment:
I think both pages should use the exact same component. Too much duplicates in this component. So we just have a piece of state that tells us if we're editing or creating, and use that to display data or information accordingly.